Data/SQLite/cmake/PocoDataSQLiteConfig.cmake

```diff
@@ -2,3 +2,10 @@ include(CMakeFindDependencyMacro)
 find_dependency(PocoFoundation)
 find_dependency(PocoData)
 include("${CMAKE_CURRENT_LIST_DIR}/PocoDataSQLiteTargets.cmake")
+
+if (@POCO_UNBUNDLED@)
+    if (@HUNTER_ENABLED@)
+        hunter_add_package(sqlite3)
```

Comment:
`hunter_add_package` not needed here.

Reply:
Removed: https://github.com/hunter-packages/poco/pull/1/commits/056037f3dbcdf9ba5a2bbde3fd219e4f521df7ea